Sample Data/ASIM/CommonSecurityLog_Schema.csv

```diff
@@ -0,0 +1,161 @@
+﻿ColumnName,ColumnOrdinal,DataType,ColumnType
+TenantId,0,"System.String",string
+TimeGenerated,1,"System.DateTime",datetime
+DeviceVendor,2,"System.String",string
+DeviceProduct,3,"System.String",string
+DeviceVersion,4,"System.String",string
+DeviceEventClassID,5,"System.String",string
+Activity,6,"System.String",string
+LogSeverity,7,"System.String",string
+OriginalLogSeverity,8,"System.String",string
+AdditionalExtensions,9,"System.String",string
+DeviceAction,10,"System.String",string
+ApplicationProtocol,11,"System.String",string
+EventCount,12,"System.Int32",int
+DestinationDnsDomain,13,"System.String",string
+DestinationServiceName,14,"System.String",string
+DestinationTranslatedAddress,15,"System.String",string
+DestinationTranslatedPort,16,"System.Int32",int
+CommunicationDirection,17,"System.String",string
+DeviceDnsDomain,18,"System.String",string
+DeviceExternalID,19,"System.String",string
+DeviceFacility,20,"System.String",string
+DeviceInboundInterface,21,"System.String",string
+DeviceNtDomain,22,"System.String",string
+DeviceOutboundInterface,23,"System.String",string
+DevicePayloadId,24,"System.String",string
+ProcessName,25,"System.String",string
+DeviceTranslatedAddress,26,"System.String",string
+DestinationHostName,27,"System.String",string
+DestinationMACAddress,28,"System.String",string
+DestinationNTDomain,29,"System.String",string
+DestinationProcessId,30,"System.Int32",int
+DestinationUserPrivileges,31,"System.String",string
+DestinationProcessName,32,"System.String",string
+DestinationPort,33,"System.Int32",int
+DestinationIP,34,"System.String",string
+DeviceTimeZone,35,"System.String",string
+DestinationUserID,36,"System.String",string
+DestinationUserName,37,"System.String",string
+DeviceAddress,38,"System.String",string
+DeviceName,39,"System.String",string
+DeviceMacAddress,40,"System.String",string
+ProcessID,41,"System.Int32",int
+EndTime,42,"System.DateTime",datetime
+ExternalID,43,"System.Int32",int
+ExtID,44,"System.String",string
+FileCreateTime,45,"System.String",string
+FileHash,46,"System.String",string
+FileID,47,"System.String",string
+FileModificationTime,48,"System.String",string
+FilePath,49,"System.String",string
+FilePermission,50,"System.String",string
+FileType,51,"System.String",string
+FileName,52,"System.String",string
```

Comment:
Please set FileName

Reply:
It's already available but columnName of Sourcetable is same as parser column. Hence we haven't mentioned specifically in parser code.